src/test/kotlin/racing/CarRacingTest.kt

```diff
@@ -7,13 +7,82 @@ class CarRacingTest {
     @Test
     fun `레이싱을 해본다`() {
         val carRacing = CarRacing()
-        carRacing.moveCars(cars = listOf(Car(), Car(), Car()), moveCount = 5)
+        carRacing.moveCars(cars = listOf(Car("현대"), Car("KIA"), Car("SAMSUNG")), moveCount = 5)
     }
 
     @Test
-    fun `차 리스트 생성`() {
+    fun `차 리스트 생성하고 차 리스트에 각자 이름이 제대로 들어간다`() {
         val carRacing = CarRacing()
-        val cars = carRacing.createCars(5)
-        assertThat(cars.size).isEqualTo(5)
+        val cars: List<Car> = carRacing.createCars(listOf("현대", "samsung", "L9"))
+        assertThat(cars[0].name).isEqualTo("현대")
+        assertThat(cars[1].name).isEqualTo("samsung")
+        assertThat(cars[2].name).isEqualTo("L9")
```

Comment:
```suggestion
        assertThat(cars).containsExactlyInAnyOrder(Car("현대"), Car("samsung"), Car("L9"))
```
객체비교를 하면 좀 더 간결한 코드를 얻을 수 있습니다. 이 코드가 동작할 수 있도록 개선해 봅시다.